src/fastapibootcamp/domain/models.py

```diff
@@ -10,7 +10,9 @@
 from astropy.coordinates import AltAz, Angle, SkyCoord
 from astropy.time import Time
 
-__all__ = ["Observer", "TargetObservability"]
+from fastapibootcamp.dependencies.pagination import Pagination
```

Comment:
Minor nit: Do you want to use a relative import here? I usually do and it looked like you were elsewhere, although I didn't know if there was a reason to avoid it here.